### DISCUSSION TITLE
How to configure for regulary backup and prune jobs?

### DISCUSSION THREAD

**Discussion by jofland** [djmaze/resticker#152] [Q&A] [answered]
I understood that the parameters `BACKUP_CRON` and `PRUNE_CRON` are mutually exclusive and cannot be configured in the docker container at the same time.

How can I configure the service to backup each night and do regular prune jobs (eg once a week)?

**Comment by ThomDietrich** [comment, marked as answer]
You define two independent instances. An example is given here:
https://github.com/djmaze/resticker/blob/master/docker-compose.example.yml

**Reply by jofland** [reply]
I feared of this answer. I will try it.
